README.md

```diff
@@ -40,34 +36,42 @@ This project is built to demonstrate how to build Orders Microservices applicati
 
 ### How it works
 
-Orders Microservice serves 'IBM Cloud Native Reference Architecture' suite, available at
-https://github.com/ibm-cloud-architecture/refarch-cloudnative-kubernetes. Though it is a part of a bigger application, Order service is itself an application in turn that persists the data of orders to a MYSQL database.
+The Orders MicroService serves 'IBM Cloud Native Reference Architecture' suite, available at
+https://github.com/ibm-cloud-architecture/refarch-cloudnative-kubernetes. Though it is a part of a bigger application, 
+the Orders service is itself an application that persists the data of orders to a MYSQL database.
```

Comment:
Isn't it a single line?

Reply:
Divided to fit 80 character ruler as a preference.
Doesn't affect outward appearance on the page.